engine/src/main/java/com/redhat/cloud/notifications/templates/EmailTemplateFactory.java

```diff
@@ -38,6 +39,8 @@ public EmailTemplate get(String bundle, String application) {
                     return new Drift();
                 case VULNERABILITY:
                     return new Vulnerability();
+                case FLEET_MANAGEMENT:
```

Comment:
In https://issues.redhat.com/browse/NOTIF-502, `fleet-management` is an app from the `edge` bundle. If that is still relevant, the app declaration will need to be moved a bit in this file because right now `fleet-management` is part of the `rhel` bundle.
```
} else if (bundle.equalsIgnoreCase(EDGE)) {
    if (application.equalsIgnoreCase(FLEET_MANAGEMENT)) {
        return new FleetManagement();
    }
}
```